ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AppDelegate.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-07.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  private var _overlayWindow: UIWindow?
+
+  var window: UIWindow?
+  var viewController: MainViewController!
+  var navigationController: UINavigationController?
+
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
+    Thread.sleep(forTimeInterval: 1.0)
+
+    window = UIWindow(frame: UIScreen.main.bounds)
+    _ = overlayWindow
```

Comment:
This particular line should be removed, as it has no effect.  We're retrieving a value to just ignore it.

Helpful doc about this for us Swift newbies:  https://stackoverflow.com/questions/39627106/why-do-i-need-underscores-in-swift

Reply:
The old idiom looks a bit awkward in Swift. `overlayWindow` is a computed property (https://github.com/keymanapp/keyman/blob/136e08b2f1c66ba856443ec9c976ba3e7c22cc71/ios/keyman/Keyman/Keyman/AppDelegate.swift#L63-L73) so retrieving the value causes the initialization.